frontend/src/components/PrivacyPolicy/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { ReactElement } from "react";
+import invariant from "invariant";
+import { useRollbar } from "@rollbar/react";
+import styles from "./PrivacyPolicy.module.css";
+import beapLogo from "../../assets/beap_lab_hex_small.png";
+
+function PrivacyPolicy(): ReactElement {
+    const rollbar = useRollbar();
+    invariant(rollbar, "Rollbar context is not available");
+    rollbar.debug("Reached Privacy Policy page");
```

Comment:
Could use rollbar.info

It's the least severe option.

Reply:
Makes sense. Done